internal/pkg/api/server/templates/telemetry/telemetry.go

```diff
@@ -154,23 +156,41 @@ func ApiClientTrace() client.TraceFactory {
 			requestSpan, ctx = ddtracer.StartSpanFromContext(
 				c,
 				"kac.api.client.request",
-				ddtracer.ResourceName(clientRequest.URL()),
+				ddtracer.ResourceName(strhelper.MustUrlPathUnescape(clientRequest.URL())),
 				ddtracer.SpanType("kac.api.client"),
+				ddtracer.AnalyticsRate(1.0),
 			)
 
 			// Set tags
-			requestSpan.SetBaggageItem("kac.api.client.request.method", clientRequest.Method())
-			requestSpan.SetBaggageItem("kac.api.client.request.url", clientRequest.URL())
-			requestSpan.SetBaggageItem("kac.api.client.request.result_type", resultType)
+			requestSpan.SetTag("kac.api.client.request.method", clientRequest.Method())
+			requestSpan.SetTag("kac.api.client.request.url", strhelper.MustUrlPathUnescape(clientRequest.URL()))
+			requestSpan.SetTag("kac.api.client.request.result_type", resultType)
+			for k, v := range clientRequest.QueryParams() {
+				requestSpan.SetTag("kac.api.client.request.params.query."+k, v)
+			}
 			for k, v := range clientRequest.PathParams() {
-				requestSpan.SetBaggageItem("kac.api.client.request.params"+k, v)
+				requestSpan.SetTag("kac.api.client.request.params.path."+k, v)
 			}
 
 			return ctx
 		}
 		t.HTTPRequestStart = func(r *http.Request) {
+			// Finish retry delay span
+			if retryDelaySpan != nil {
+				requestSpan.Finish()
```

Comment:
Dole od riadku `216` som su prepisoval hodnotu callbacku `HTTPRequestStart` a cudoval som sa, preco to nefunguje ... tj. kod som skopiroval sem.